README.md

```diff
@@ -15,6 +15,20 @@ Here is the list of device backends that we support:
 - [SCSI](https://github.com/rust-vmm/vhost-device/blob/main/crates/vhost-device-scsi/README.md)
 - [VSOCK](https://github.com/rust-vmm/vhost-device/blob/main/crates/vhost-device-vsock/README.md)
 
+For `vhost-user` device backends that have no final specification merged into the specification, we have a [staging workspace](./staging/).
+More information may be found in its [README file](./staging/README.md).
+
+Here is the list of device backends in **staging**:
```

Comment:
Why duplicate this information here ? I think staging's readme should be enough.

Also it isn't only about unmerged specifications anymore, sound isn't one of them. Maybe a better description ?

Reply:
> Why duplicate this information here ? I think staging's readme should be enough.

I also prefer a single place, but I think it is better on the main README, so we have one place where we list all the crates in this repository, both the "official" ones and the staging ones.

Reply:
not a strong opinion, just a preference ;-)

Reply:
I am fine with that too, @epilys just move them at a single place :)

Reply:
I removed the list from the subdir README, and changed the desc a bit, do you think the wording is OK now?